cocos/network/CCDownloader-curl.cpp

```diff
@@ -284,10 +284,10 @@ class DownloadTaskCURL : public IDownloadTask {
     string _tempFileName;
     std::string _checksumFileName;
     vector<unsigned char> _buf;
-    FileStream* _fs = nullptr;
+    std::unique_ptr<FileStream> _fs = nullptr;
```

Comment:
Since we use unique_ptr, this line could be simplified to `std::unique_ptr<FileStream> _fs{};`

Reply:
I'll update all instances to use the `{}`.